### PR TITLE
[bitnami/postgresql] Fix loadBalancerSourceRanges in svc-read

### PR DESCRIPTION
**Pull request by juan131**
Signed-off-by: juan131 <juan@bitnami.com>

**Description of the change**

This PR ensure the `loadBalancerSourceRanges` are also taken into account for the `svc-read` service.

**Possible drawbacks**

None

**Applicable issues**

  - fixes https://github.com/bitnami/charts/issues/2114

**Checklist**
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

### CODE REVIEW

File: bitnami/postgresql/templates/svc-read.yaml [hunk 1]
```diff
@@ -8,15 +8,17 @@ metadata:
     chart: {{ template "postgresql.chart" . }}
     release: {{ .Release.Name | quote }}
     heritage: {{ .Release.Service | quote }}
-{{- with .Values.service.annotations }}
-  annotations:
-{{ toYaml . | indent 4 }}
-{{- end }}
+  {{- if .Values.service.annotations }}
```
Comment (dani8art) [inline bitnami/postgresql/templates/svc-read.yaml]: Could we add sth in documentation to understand they are evaluated as a template?

File: bitnami/postgresql/templates/svc.yaml [hunk 2]
```diff
@@ -7,20 +7,16 @@ metadata:
     chart: {{ template "postgresql.chart" . }}
     release: {{ .Release.Name | quote }}
     heritage: {{ .Release.Service | quote }}
-{{- with .Values.service.annotations }}
-  annotations:
-{{ tpl (toYaml .) $ | indent 4 }}
-{{- end }}
+  {{- if .Values.service.annotations }}
+  annotations: {{- include "postgresql.tplValue" (dict "value" .Values.service.annotations "context" $) | nindent 4 }}
```
Comment (dani8art) [inline bitnami/postgresql/templates/svc.yaml]: same here.

File: bitnami/postgresql/templates/svc.yaml [hunk 3]
```diff
@@ -7,20 +7,16 @@ metadata:
...
   type: {{ .Values.service.type }}
   {{- if and .Values.service.loadBalancerIP (eq .Values.service.type "LoadBalancer") }}
   loadBalancerIP: {{ .Values.service.loadBalancerIP }}
   {{- end }}
   {{- if and (eq .Values.service.type "LoadBalancer") .Values.service.loadBalancerSourceRanges }}
-  loadBalancerSourceRanges:
-  {{ with .Values.service.loadBalancerSourceRanges }}
-{{ toYaml . | indent 4 }}
-{{- end }}
+  loadBalancerSourceRanges: {{- include "postgresql.tplValue" (dict "value" .Values.service.loadBalancerSourceRanges "context" $) | nindent 4 }}
```
Comment (dani8art) [inline bitnami/postgresql/templates/svc.yaml]: same here

File: bitnami/postgresql/templates/svc-read.yaml [hunk 4]
```diff
@@ -8,15 +8,17 @@ metadata:
...
+  annotations: {{- include "postgresql.tplValue" (dict "value" .Values.service.annotations "context" $) | nindent 4 }}
+  {{- end }}
 spec:
   type: {{ .Values.service.type }}
   {{- if and .Values.service.loadBalancerIP (eq .Values.service.type "LoadBalancer") }}
   loadBalancerIP: {{ .Values.service.loadBalancerIP }}
   {{- end }}
+  {{- if and (eq .Values.service.type "LoadBalancer") .Values.service.loadBalancerSourceRanges }}
+  loadBalancerSourceRanges: {{- include "postgresql.tplValue" (dict "value" .Values.service.loadBalancerSourceRanges "context" $) | nindent 4 }}
```
Comment (dani8art) [inline bitnami/postgresql/templates/svc-read.yaml]: same here

**Comment by juan131** [comment]
Waiting for CI/CD to update images' tags

**Pull request merged**